src/develop/lightroom.c

```diff
@@ -469,560 +1009,109 @@ void dt_lightroom_import(int imgid, dt_develop_t *dev, gboolean iauto)
 //     return;
 //   }
 
-  xmlXPathFreeObject(xpathObj);
-  xmlXPathFreeContext(xpathCtx);
+  // let's now parse the needed data
 
-  // Go safely to Description node
+  lr_data_t data;
```

Comment:
Why all the `memset` instead of a single `lr_data_t data = {0};`?

Also, I hope it's not too big for the stack. @LebedevRI might know.

Reply:
The simplest/dumbest code is *usually* the best one.
As far as i can tell there is no difference whatsoever between `{}` vs `memset()`.
https://godbolt.org/g/qMc46p
But the `memset()` is worse because that is much more lines, symbols; and you need to specify sizes, offsets.

TLDR: just do what @houz suggested `lr_data_t data = {0};`

Reply:
Indeed, better. Fixing.

Reply:
Also I don't think there is change on the stack usage, the variables were there before.